oletools/olevba.py

```diff
@@ -1858,12 +1860,12 @@ def __init__(self, ole, vba_root, project_path, dir_path, relaxed=True):
                 break
 
             if check == 0x0016:
-                # REFERENCENAME
+                REFERENCENAME = "REFERENCENAME"
```

Comment:
Changed the comments to constants because further down I wanted not to duplicate a string.